core/src/main/java/org/jdbi/v3/core/statement/BindListEmptyHandling.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.statement;
+
+import java.util.function.BiConsumer;
+
+/**
+ * describes what should be done if the value passed to {@link SqlStatement#bindList} is null or empty
+ */
+public enum BindListEmptyHandling implements BiConsumer<SqlStatement, String> {
```

Comment:
Should this really have `BindList` in the name?  What if we want to use it from another feature?  Seems like this might be overly specific.

Reply:
I named it so specifically so that it would be discoverable via IDE hints to people typing "bindlist", and because it really doesn't feel like something that can be reused for anything. But if you insist

Reply:
I don't insist.  Just a comment.  I don't have a strong preference.